src/components/summary/styles.scss

```diff
@@ -1,16 +1,14 @@
 .c-summary {
 
-  li {
-    display: flex;
-    flex-direction: column;
-    align-items: center;
-    color: $color-3;
-    font-family: $font-family-1;
-    font-weight: bold;
+  article {
```

Comment:
I've strongly recommended you use class names instead of use element names in CSS as much as possible. If you don't want... use BEM at least:

```
.c-summary {
  > article {} 
}
```